amy/workshops/tests/test_person.py

```diff
@@ -1384,7 +1384,18 @@ def setUp(self):
             password="pass",
         )
         self.user.data_privacy_agreement = True
+        self.user.github = "user-github"
+        self.user.is_active = True
         self.user.save()
+        self.user.synchronize_usersocialauth()
```

Comment:
Looks like this would call the GitHub API for [this user](https://github.com/user-github). Can you mock the underlying `Github` class (see `github_auth.py` for details)?

Reply:
Gotcha added it.